src/trumbowyg.js

```diff
@@ -608,6 +609,24 @@ jQuery.trumbowyg = {
                     if (e.type === 'blur') {
                         $('.' + prefix + 'active-button', t.$btnPane).removeClass(prefix + 'active-button ' + prefix + 'active');
                     }
+                    if (t.o.autogrowOnEnter) {
+                        if (e.type === 'focus') {
+                            t.autogrowOnEnter_hasHadFocus = true;
```

Comment:
This name does not fit other var names of the project :/

You can just name that `autogrowOnEnterWasFocused`